setup.py

```diff
@@ -41,21 +41,23 @@
 ]
 
 install_requires = [
-    'adage==0.8.5',
+    'adage==0.10.0',
     'celery>=4.1.0,<4.3',
     'click>=7',
     'enum34>=1.1.6',
-    'packtivity==0.10.0',
-    'pyOpenSSL==17.5.0',  # FIXME remove once yadage-schemas solves deps.
+    'jq==0.1.6',
+    'networkx==2.3',
+    'packtivity==0.14.21',
+    # 'pyOpenSSL==17.5.0',  # FIXME remove once yadage-schemas solves deps.
     'reana-commons>=0.6.0.dev20190619,<0.7.0',
-    'requests==2.20.0',
-    'rfc3987==1.3.7',  # FIXME remove once yadage-schemas solves deps.
-    'strict-rfc3339==0.7',  # FIXME remove once yadage-schemas solves deps.
+    'requests==2.22.0',
+    # 'rfc3987==1.3.7',  # FIXME remove once yadage-schemas solves deps.
```

Comment:
If these are not needed anymore, great, please remove them from everywhere instead of commenting... The git history is our friend, so if we need to go back, we can.  There is no need for commented code in the live code base...